fixtures/yuer/retro-6060.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Retro 6060",
```

Comment:
Is this the "[6x60](https://www.yuerlights.com/products/nordic-6x60w-led-retro-burst-flashing-night-light-dmx512-2200k-led-flash-stage-dj-party-photographic-effect-lighting-equipment?variant=45464628199702)"?  I can't find anything called the Yuer Retro 6060.